tests/foreman/cli/test_contentaccess.py

```diff
@@ -15,251 +15,160 @@
-from robottelo.test import CLITestCase
-from robottelo.vm import VirtualMachine
-
-
-@pytest.mark.run_in_one_thread
-class ContentAccessTestCase(CLITestCase):
-    """Content Access CLI tests."""
-
-    @classmethod
-    @skip_if_not_set('clients', 'fake_manifest')
-    def setUpClass(cls):
-        """Setup must ensure there is an Org with Golden Ticket enabled.
-
-
-        Option 1) SQL::
-
-            UPDATE
-                 cp_owner
-            SET
-                 content_access_mode = 'org_environment',
-                 content_access_mode_list='entitlement,org_environment'
-            WHERE account='{org.label}';
-
-        Option 2) manifest::
-
-            Change manifest file as it looks like:
-
-                Consumer:
-                    Name: ExampleCorp
-                    UUID: c319a1d8-4b30-44cd-b2cf-2ccba4b9a8db
-                    Content Access Mode: org_environment
-                    Type: satellite
-
-        :steps:
-
-            1. Create a new organization.
-            2. Use either option 1 or option 2 (described above) to activate
-               the Golden Ticket.
-            3. Create a Product and CV for org.
-            4. Add a repository pointing to a real repo which requires a
-               RedHat subscription to access.
-            5. Create Content Host and assign that gated repos to it.
-            6. Sync the gated repository.
-        """
-        super().setUpClass()
-        # Create Organization
-        cls.org = make_org()
-        # upload organization manifest with org environment access enabled
-        cls.manifest = manifests.clone(org_environment_access=True)
-        manifests.upload_manifest_locked(
-            cls.org['id'], cls.manifest, interface=manifests.INTERFACE_CLI
-        )
-        # Create repositories
-        cls.repos = [
-            # Red Hat Satellite Tools from CDN
-            {
-                'product': PRDS['rhel'],
-                'repository-set': REPOSET['rhst7'],
-                'repository': REPOS['rhst7']['name'],
-                'repository-id': REPOS['rhst7']['id'],
-                'releasever': REPOS['rhel7']['releasever'],
-                'arch': REPOS['rhel7']['arch'],
-                'cdn': True,
-            },
-        ]
-        cls.repos_info = setup_cdn_and_custom_repositories(cls.org['id'], cls.repos)
-        # Create a content view
-        content_view = make_content_view({'organization-id': cls.org['id']})
-        # Add repositories to content view
-        ContentView.add_repository(
+from robottelo.config import settings
+from robottelo.constants import REAL_0_ERRATA_ID
+from robottelo.constants import REAL_RHEL7_0_2_PACKAGE_FILENAME
+from robottelo.constants import REAL_RHEL7_0_2_PACKAGE_NAME
+
+
+pytestmark = [
+    pytest.mark.skipif((not settings.repos_hosting_url), reason='Missing repos_hosting_url'),
+    pytest.mark.run_in_one_thread,
+]
+# """Content Access CLI tests."""
```

Comment:
Optional - This was from the class docblock, its a bit out of place here.  L1 of the module docblock states as much, so I don't think this comment provides much additional context.

Reply:
OK, removed